interface_config.js

```diff
@@ -4,6 +4,7 @@ var interfaceConfig = {
     // TO FIX: this needs to be handled from SASS variables. There are some
     // methods allowing to use variables both in css and js.
     DEFAULT_BACKGROUND: '#474747',
```

Comment:
Isn't the TO FIX comment above necessary? Doesn't look like it.

Reply:
Indeed, I removed it.